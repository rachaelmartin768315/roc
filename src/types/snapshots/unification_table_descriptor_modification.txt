New type variable: {FlexVar(null), rank: 0, mark: NONE, copy: null}
==================== Unification Table ====================
Var 0: {FlexVar(null), rank: 0, mark: NONE, copy: null}
===========================================================
INFO: Modify descriptor change Mark from NONE to OCCURS
INFO: Changing Mark from NONE to OCCURS
INFO: Changing Rank from 0 to 1
==================== Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: OCCURS, copy: null}
===========================================================
