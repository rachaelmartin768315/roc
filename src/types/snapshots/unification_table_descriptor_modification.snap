New type variable: {FlexVar(null), rank: 0, mark: NONE, copy: null}
==================== START Unification Table ====================
Var 0: {FlexVar(null), rank: 0, mark: NONE, copy: null}
==================== END Unification Table ====================
INFO: Modify descriptor change Mark from NONE to OCCURS
INFO: Changing Mark from NONE to OCCURS
INFO: Changing Rank from 0 to 1
==================== START Unification Table ====================
Var 0: {FlexVar(null), rank: 1, mark: OCCURS, copy: null}
==================== END Unification Table ====================
